R/borrowing_details.R

```diff
@@ -1,11 +1,18 @@
 #' Legacy function for specifying borrowing details
 #'
-#' Please use one of `hierarchical_commensurate_borrowing()`, `no_borrowing()`, or `full_borrowing()` instead.
+#' Please use one of `borrowing_hierarchical_commensurate()`, `borrowing_none()`, or `borrowing_full()` instead.
```

Comment:
```suggestion
#' Please use one of [borrowing_hierarchical_commensurate()], [borrowing_none()], or [borrowing_full()] instead.
```
I think making them links is even better so we get errors if the links don't work.